src/database/models/eventsModel.js

```diff
@@ -1,8 +1,8 @@
-var mongoose = require('mongoose');
-var Schema = mongoose.Schema;
+let mongoose = require('mongoose');
+let Schema = mongoose.Schema;
 
 //Event model consisting of the event title, description, start and end time, checkIn code, and splash URL
-var eventSchema = new Schema(
+let eventSchema = new Schema(
```

Comment:
Good call to switch away from let - why don't we just switch all our variable types to const?